src/main/java/com/minecolonies/coremod/entity/CustomArrowEntity.java

```diff
@@ -17,7 +17,7 @@
     /**
      * Max time the arrow is stuck before removing it
      */
-    private static final int MAX_TIME_IN_GROUND = 10;
+    private static final int MAX_TIME_IN_GROUND = 100;
 
```

Comment:
How about we init an arrow with a custom time, so for the arrows of our trainees we can make this 100, and all other stays 10?

Reply:
hm dont think we need that, those 5 sec should be still fine for all needs

Reply:
Oh the motivation is more to have it remove them quicker if we don't need them 5s. (Even less lag)

Reply:
ye doesnt make a difference really, both despawn rather quick